src/components/calcite-date-month-header/calcite-date-month-header.tsx

```diff
@@ -89,14 +89,18 @@ export class CalciteDateMonthHeader {
     return (
       <Host dir={dir}>
         <div aria-hidden="true" class="header">
-          <button
-            aria-label={this.intlPrevMonth}
+          <a
```

Comment:
Can you keep the element type as `button`? Seems like it would be more semantic rather than a link with a role and a href.

Reply:
It seems like the preventDefault would be enough with the button element right?

Reply:
No, that's the whole fix is _not_ using a button.

Reply:
This _may_ be a Stencil bug. 

`blur` shouldn't bubble, but for some reason it's reaching the host event handler.

```tsx
import { Component, Host, h } from "@stencil/core";

@Component({
  tag: "calcite-test",
  shadow: true
})
export class CalciteTest {
  handleHostBlur(event: FocusEvent): void {
    console.log("host blur", event);
  }

  handleInputBlur(event: FocusEvent): void {
    console.log("input blur", event);
  }

  render() {
    return (
      <Host onBlur={this.handleHostBlur}>
        <input onBlur={this.handleInputBlur} placeholder="focus here first" />
        <button>then click here</button>
      </Host>
    );
  }
}
```

The host event handlers don't fire in a non-Stencil custom element:

```js
customElements.define("blur-test",
  class BlurTest extends HTMLElement {
    connectedCallback() {
      this.attachShadow({
        mode: "open"
      });

      this.shadowRoot.innerHTML = `
    <div id="fake-host">
      <input id="input" placeholder="focus here first!" />
      <button>then click here</button>
    </div>
    `

      this.shadowRoot.getElementById("fake-host")
        .addEventListener("blur", (event) => {
          console.log("host caught blur", event);
        });

      this.shadowRoot.getElementById("input")
        .addEventListener("blur", (event) => {
          console.log("input caught blur", event);
        });
    }
  })
```

Reply:
@jcfranco quite possible. So you're basically saying this code never should have run at all? Ie. host blur should never have fired in any scenario?

Reply:
> host blur should never have fired in any scenario?

That's my expectation.

Reply:
FYI https://github.com/ionic-team/stencil/issues/2699

Reply:
Nice thanks for creating that.